src/TodoListItem.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+// import TodoList from "./TodoList";
+
+
+
+const TodoListItem = (props) => {
+    return(
+        <li key={props.key}> {props.todo} </li>
+    )
+};
+
+
+
+// const Search = () => {
+
+// ///ha
+// const handleChange = (event) => {
+// console.log(event.target.value);
+// };
+
+// return (
+//     <div>
+//         <label htmlFor="search"> Search: </label>
+//         <input id="search" type="text" onChange={handleChange}></input>
+//     </div>
+// );
+
+// };
```

Comment:
same here we don't want to have any unrelated code in the PR, PR should include only the code it is meant to be